bin/oneoffs/remove_cas.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+import datetime
+import logging
+import os
+import shutil
+import uuid
+
+from collections import Counter
+
+from api import config
+from api import util
+
+
+logging.basicConfig(
+    format='%(asctime)s %(name)16.16s %(filename)24.24s %(lineno)5d:%(levelname)4.4s %(message)s',
+    datefmt='%Y-%m-%d %H:%M:%S',
+    level=logging.DEBUG,
+)
+log = logging.getLogger('remove_cas')
+
+
+def get_files_by_prefix(document, prefix):
+    for key in prefix.split('.'):
+        document = document.get(key, {})
+    return document
+
+
+def copy_file(path, target_path):
+    target_dir = os.path.dirname(target_path)
+    if not os.path.exists(target_dir):
+        os.makedirs(target_dir)
+    shutil.copy(path, target_path)
+
+
+def remove_cas():
+    """
+    Remove CAS logic, generate UUID for the files and rename them on the filesystem, make a copy of the file if more
+    than one container using the same hash.
+    """
+    COLLECTIONS_PREFIXES = [('projects', 'files'),
```

Comment:
@davidfarkas Are collection attachments (the scitran kind, not mongo) accounted for?

Reply:
Nice catch

Reply:
Just tested collection attachments, and they do get migrated. disregard